### ISSUE TITLE
HAL Deposit Status 'update'

### ISSUE THREAD

**Issue by beckstefan** [dissemin/dissemin#869]
Currently we have a roundabout 80 deposits on HAL with "permanent" status `pending`, i.e. the status did not change for a long time.

When investigating the SWORD response, we see them classified as `update`. The HAL documentation says:

> update : une(des) modification(s) a(ont) été demandée(s) lors de la validation

In the comment, there is some human written text what to update.

I am not not an expert concerning HAL, but we should do something about this. Ideally we or HAL can give the feedback to the depositors and they can edit the entry accordingly.

**Comment by wetneb** [comment]
Yes, generally these deposits have been put on hold by the moderators because the PDF supplied looks like a published copy. So we should withdraw these deposits, perhaps?

**Comment by beckstefan** [comment]
Yes, this is an option to. Would you just delete them silently, set to `failed` or introduce new status?

**Comment by wetneb** [comment]
Ideally we probably should mark them as abandoned in HAL directly, to clear the queue on their side, but I have no idea if this can be done programmatically (or if it is possible to abandon a HAL deposit at all). Perhaps they could be considered as `stale` after a while (and we would not seek to update the status of `stale` deposits)?

**Comment by beckstefan** [comment]
We can remove them from HAL: https://api.archives-ouvertes.fr/docs/sword#delete

Sorry, I had that in mind when writing my above post.

**Comment by wetneb** [comment]
That would probably make sense then!

**Comment by beckstefan** [comment]
So, if someone tackles this issue:

* Remove from HAL
* Set status to `refused`
